pretixprint/app/src/main/java/com/neostra/interfaces/INeostraInterfaces.java

```diff
@@ -0,0 +1,1352 @@
+package com.neostra.interfaces;
+
+import android.os.Binder;
+import android.os.IBinder;
+import android.os.IInterface;
+import android.os.Parcel;
+import android.os.RemoteException;
+
+public interface INeostraInterfaces extends IInterface {
```

Comment:
As discussed, if we can, I think we could delete the methods we certainly don't need.